lib/cdo/script_config.rb

```diff
@@ -31,6 +31,8 @@ def self.scale_mode_scripts
       coursed
       coursee
       coursef
+      express
+      pre-express
```

Comment:
Is this supposed to include _all_ of our Hour of Code tutorials or did we exclude our high ticket Hour of Code tutorials on purpose?

High traffic tutorials:
- `mc`
- `minecraft`
- `flappy`
- `starwars`
- `starwarsblocks`
- `frozen`
- `hourofcode`

Not so high traffic tutorials that are missing from the list:
- `gumball`
- `sports`
- `basketball`
- `text-compression`
- `hoc-encryption`

Reply:
@tanyaparker the other scripts come from a different list, `cached_scripts`. That list and this one (that I added `express` and `pre-express` to) are combined here: 

```
  def self.scale_mode_scripts
    cached_scripts + %w(
      20-hour
      course1
      ...
```

`cached_scripts` comes from `CACHED_SCRIPTS_MAP` here: https://github.com/code-dot-org/code-dot-org/blob/staging/cookbooks/cdo-varnish/libraries/http_cache.rb#L19

That contains all the scripts you were asking about except `text-compression` and `hoc-compression`. I suspect because those are not used at high scale during HOC. Since we haven't marked those two as scale scripts in the past I think we're OK on that.

Reply:
Sounds good!